### DISCUSSION TITLE
【PgFlow】テンプレート募集

### DISCUSSION THREAD

**Discussion by snext1220** [snext1220/stext#264] [Show and tell]
#185 のIssueより派生しております。
PgFlow（PgEditor）で「こんなテンプレがあればいいな」的なご意見をよく見かけるようになりましたので、その登録用Issueです。/Playground/template フォルダーに.jsonファイルをUpした上で、以下の項目をお知らせ戴けますと幸いです。

+ テンプレート名（5～9文字程度）
+ ファイル名
+ 趣旨（こんな目的で使ってほしい、既存テンプレとの違い、対象製作者など）

あまりに似たようなテンプレートが増えてきた場合にはルール決めも必要になるかもしれませんが、まずはあまり難しく考えずに必要なものをUp戴ければ、こちらにて登録させていただきます。宜しくご協力戴けますと幸いですm(_ _)m

**Comment by toki-sor1** [comment]
大変お待たせしました…！
短編用テンプレートが完成しましたので、/Playground/template フォルダーにアップしました！

- テンプレート名： 短編30改変用
- ファイル名： mini30
- 趣旨：

Scene 0～30まで予め作成済みのテンプレートです。

このテンプレート独自のサンプルデータも入力済みで、基本的なシナリオ作成の流れも解説しています。
（何がどのデータに関係しているのか分かり易く…を心掛けました）

最初はシナリオ作成の経験者向けのものを想定していたのですが、解説を入れている内に未経験者向けになりました。
「短編シナリオを書いてみたいけど、Pg Flowの使い方が分からない」、「ゲームブックのシナリオを書くのは初めてで、書き方がよく分からない」というユーザーさん向けでしょうか。

Scene 5～27はリンクを設定していないので、自由にリンクを消したり繋いだりしていただければいいな～、と。

**Comment by snext1220** [comment]
改めてテンプレートのUploadありがとうございます！
先ほど本番環境にも反映させていただきました～

ゲームブックのシナリオを書くのは初めて＞はい～テンプレートを使う方は初心者が多いと思うので、方針賛成です！
＃中級者以上向けのテンプレートもあってよいとは漠然と思うのですが、どんなのが必要なのか想像できないｗ

以下気づいた点です。

+ Scene 5～27：現在のフロチャ表示ですと、こちらの表示に引きずられて、本来のツリーが初期状態で欄外になってしまうのがやや気になりました。フリーのSceneを減らす？
+ Tips 2：今回新たにタブからの削除機能もできましたし、https://github.com/snext1220/stext/issues/207#issuecomment-663854455 での機能削除の話題も出ていますので、タブからの削除をご紹介戴いた方が良いかもしれません（？）

あと、今回テンプレを拝見して思ったのですが、タブ冒頭の説明にもクリックでマニュアル表示は書いておいた方が良いのかもしれませんね（現在も起動時のトーストでは表示されはするのですが^^;）

**Comment by snext1220** [comment]
本件、テンプレートに対して、拙作の三本を入れてみました。
初級者向けだけでなく、実践的なサンプルで学んでもらおうという意図です。
ネタバレにもつながるため、テンプレートとして載せても良いよ、というシナリオのみ入れてまいりますので、もし掲載可能な方はお申し付けくださいm(_ _)m

伴い、選択ボックス上、テンプレートの分類を追加しました。
分類名については一寸イマイチな感じもあるので、ベター案あればお願いします～

**Comment by cocotori** [comment]
チュートリアル的なテンプレートもいいですが、
実際のシナリオのテンプレートがあるといいですね！
なるほど、こうやって作ってるのか、というのがわかりやすくなって
とてもいいと思います。

**Comment by toki-sor1** [comment]
お返事が遅くなってしまい申し訳ありません…！
https://github.com/snext1220/stext/issues/194#issuecomment-663859901 では、ご確認＆ご助言ありがとうございました！
ご助言を参考に、Scene数を15に改めてフロチャを見易くし、リンク（シーン）削除の辺りも修正いたしました。
/Playground/template フォルダーにアップ済みです。

シーン数変更に伴い、ファイル名も「mini15」と変更しております。
お手数をおかけしてしまい恐縮ですが、本番環境にもアップ済みですので、 **「mini30」の削除と「mini15」の追加**をお願いいたします；；

> 実際のシナリオ、分類名

GBを遊ぶことで構造が気になるという御方（主に作家さん？）もいらっしゃると思いますので、とても良いと思います！
分類名もテンプレートの種類が分かりやすくて良いと思います！

**Comment by snext1220** [comment]
修正＆Updateありがとうございます！
こちらPgFlowにも反映いたしましたので、ご確認戴けますと幸いです。

> 実際のシナリオ、分類名

こちらも問題ないようで良かったです！
なお、実践サンプルについては、PgFlow導入前のシナリオであることからsummaryなしのものばかりになってしまっているのですが、もし可能であればmarsh、usherなどをテンプレとして追加させていただくことは可能でしょうか（勿論、ネタバレなどで抵抗ある可能性もあるので、難しいようであればスルーしてください）。

その他、必要なテンプレについては引き続き募集中なので、引き続きよろしくお願いいたします＞皆さま

**Comment by cocotori** [comment]
反応遅くなり申し訳ありません。
導きソーサリアンは、テンプレに加えてもらって問題ありません。

**Comment by snext1220** [comment]
@cocotori さん：
ご了解有難うございます～
先ほどPgFlowに追加させて戴きました！（GitHub Only）

皆さん：
本Issueについては、とりあえずMonthly Milestoneとしては一旦の役割を終えましたので、次回Jan Updateからは外してまいりますね。Issueとしては残しておくので、追加テンプレートを作成されたなどの場合は、引き続きこちらから投稿戴けますと幸いです。
